composer.json

```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": "^5.3.9|^7.0",
-        "imagine/Imagine": "^0.6.3,<0.7",
+        "imagine/Imagine": "^0.6.3|^0.7.0,<0.8",
```

Comment:
> It's important to note that the rule you mentioned only applies for fully enumerated versions

I meant to append that with "above `1.x`" and I obviously used the wrong example of `0.7` where I intended to use `1.0`. Was typing quickly. ;-)